### ISSUE TITLE
Allow handling of `iOS` Pen double taps / squeezes

### ISSUE THREAD

**Issue by ActuallyHappening** [rust-windowing/winit#3759]
### Description

I am trying to implement a 100% Rust note taking application for my iPad which can integrate with my Apple Pencil, which supports double tapping the tip of the pencil. This functionality is **critical** for apps to handle using Apple Pencil, and I would very much like this functionality to be added to `bevy` somehow since that is the library I am building on top of.
I believe adding this as a `platform_impl/iOS` platform feature would be sufficient, since I can `#[cfg(feature = "iOS")]` any code depending on it away anyway.

I have no experience with `objc2`, `icrate`, `UIKit` or even `winit` itself yet, and would definitely like some pointers (pun) to get me started.

### Relevant platforms

iOS

**Comment by daxpedda** [comment]
Isn't this covered by #99?
Or why open a separate issue?

**Comment by ActuallyHappening** [comment]
> Isn't this covered by #99? Or why open a separate issue?

Yes this is a subset, I've already read #99 but it looked far broader, with "eventually Apple Pencil" though I only really care about Apple Pencil for the moment. Support for other pens is a bonus, and would be nice, but Apple Pencil for my specific use case is the only hardware I have available at the moment.

I have an idea how to implement this, using a delegate (https://developer.apple.com/documentation/applepencil/handling-double-taps-from-apple-pencil) and somehow integrating with the `objc2` (https://docs.rs/objc2/0.5.2/objc2/macro.declare_class.html) using a protocol/trait implementation like:
```rust
	// Not sure if this is all we need to do
        unsafe impl UIPencilInteractionDelegate for WinitViewController {
    #[method(pencilInteraction:didReceiveTap:)]
    fn pencilInteraction_didReceiveTap(
        &self,
        interaction: &UIPencilInteraction,
        tap: &UIPencilInteractionTap,
    ) {
        tracing::error!(
            "Pencil Double Tap, interaction: {:?}, tap: {:?}",
            interaction,
            tap
        );
    }
```

How do I propagate this as an event? I'll work that out soon.
How can you patch this into bevy? I'll work that out soon.

I don't have a PR, but my GitHub fork has quite a few `testing-*` branches

**Comment by daxpedda** [comment]
I'm going to close this in favor of #99 then.
We can re-open this if for some reason this feature is still missing after #99 is closed.
